src/odemis/gui/conf/util.py

```diff
@@ -929,6 +929,165 @@ def cb_get(ctrl=value_ctrl, va=va, u=unit):
     return setting_entry
 
 
+def create_local_axis_entry(container, name, va, comp, conf=None):
```

Comment:
That's a lot of new code, almost identical to the other ones. I *really* would like to avoid this. It should be possible to use `create_setting_entry()`, with `hw_comp=None`. If it doesn't work out-of-the-box, only minor tweaks should be needed (and with `hw_comp=None`, it's easy to distinguish).

Reply:
I tried this at first, but there are a lot of differences in the axis that are slightly different, so the result is merging the "create_axis_entry" and "create_setting_entry" if this is ok. The very large function before became rather confusing, and I was worried I'd break something.

Reply:
Leave `create_axis_entry()` as-is. However, `create_local_axis_entry()` and `create_setting_entry()` should be merged. I'm pretty convinced that the result is just the current `create_setting_entry()`, with maybe a few tweaks, if I'm not mistaken.

Reply:
Maybe... although there are a lot of tweaks it seems for the axis. I will see if I can merge it again. I think i have to change some of the ways it handles arguments - there wouldn't be any settings entries with  hw_comp=None that don't use the axes?

Reply:
Ok - while testing - apparently there are cases where hw_comp is None that are not related to local axis (_add_hw_setting_controls, with CameraStream.emitter) so is there another way to determine when the SettingEntry should default to localAxis mode?

Reply:
Also the resolution VA

Reply:
Could you point me to where is it important for the code to know that it's a "local axis" instead of a standard VA?
I'd expect that at this point, the GUI doesn't care. It should all be hidden away by the stream. Eventually it's a "setting". Some settings are properties of the hardware, some involve moving an axis, but that doesn't matter.

Reply:
I think it is just related to the way the axes are formatted